Kickstarter-iOS/Views/Controllers/PledgeTableViewController.swift

```diff
@@ -67,6 +73,14 @@ class PledgeTableViewController: UITableViewController {
         }
       }
 
+    self.viewModel.outputs.presentShippingRules
+      .observeForUI()
+      .observeValues { [weak self] project, shippingRules, selectedShippingRule in
+        self?.presentShippingRules(
+          project, shippingRules: shippingRules, selectedShippingRule: selectedShippingRule
```

Comment:
The argument naming here is fine because `selectedShippingRule` is a noun in this case, if we changed this to `selected:` even though we have the `ShippingRule` type it would change the meaning to be an action.